### PR TITLE
Maven-Surefire-Plugin 3.0.0.M5

### PR DESCRIPTION
**Pull request by mickaelistria**
To get better JUnit 5 support

### CODE REVIEW

File: tycho-surefire/org.eclipse.tycho.surefire.osgibooter/build.properties [hunk 1]
```diff
@@ -12,7 +12,7 @@ bin.includes = plugin.xml,\
                .,\
                META-INF/,\
                jars/,\
-               jars/maven-failsafe-plugin-2.22.2.jar
+               jars/maven-failsafe-plugin-3.0.0-M5.jar
```
Comment (joeshannon) [inline tycho-surefire/org.eclipse.tycho.surefire.osgibooter/build.properties:15]: Is this entry required given that the `jars/` directory is already included?

**Comment by laeubi** [comment]
FYI @mickaelistria M6 is currently voted to be released.

**Comment by laeubi** [comment]
> To get better JUnit 5 support

What recently came into my mind:

Maybe we could rename the current tycho-surefire to tycho-surefire-legacy and keep that as is with surefire2+junit4/5 and make the new surefire with surfire3 but junit5 only? junit5 in general has better means of controlling the test execution and with vintage one could also execute junit4...

**Comment by laeubi** [comment]
FYI I created https://issues.apache.org/jira/browse/SUREFIRE-2054 so we probably could somewhere later simply implement a special Starter that could be plugged inside surefire without the need to maintain surefire dependency at tycho...

**Comment by laeubi** [comment]
> The Apache Maven team is pleased to announce the release of the Apache
> Maven Surefire Plugin, version 3.0.0-M6.
> 
> The release contains 111 bug fixes.
> Again we received contributions from the community in the form of bug
> reports and bug fixes.
> Thank you and keep them coming!
> 
> https://maven.apache.org/surefire/index.html

**Comment by mickaelistria** [comment]
Opened https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/2321 for the license issue.

**Pull request merged**